.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Send Feedback
+    url: https://scratchaddons.com/docs/faq/#what-can-i-do-if-i-find-a-problem
+    about: Trying to leave feedback? Go here to learn more.
```

@Hans5958 Something like this?
```suggestion
    about: Want to say something about Scratch Addons? Learn more about leaving feedback here.
```

---

I was thinking to reword the mentions of "Send Feedback" to something like "Contact us" or something more general.

---

> I was thinking to reword the mentions of "Send Feedback" to something like "Contact us" or something more general.

Would you mind writing a potential rewording?

Personally, "contact us" isn't the only thing I'd say because we have public areas where you can leave feedback.

---

Something like "Want to say something about Scratch Addons? Tell us here!" or "Want to say something about Scratch Addons? Learn more about how to reach/tell/contact us!"

---

> "Want to say something about Scratch Addons? Learn more about how to reach/tell/contact us!"

I'll commit that.